org.eclipse.scanning.test/src/org/eclipse/scanning/test/points/RasterTest.java

```diff
@@ -188,7 +188,7 @@ public void testFillingAMoreComplicatedBoundingRectangle() throws Exception {
 		assertEquals("Point list size should be correct", rows * cols, pointList.size());
 
 		// Check some points
-		assertEquals(new Point(0, xStart, 0, yStart), pointList.get(0));
+		assertEquals(new Point(0, xStart, 0, yStart, 0), pointList.get(0));
 		assertEquals(xStart + 3 * xStep, pointList.get(3).getValue("X"), 1e-8);
 		// TODO more
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1135)